### PR TITLE
Sarich/add blues

### PR DESCRIPTION
**Pull request by sarich**

### CODE REVIEW

File: cime_config/acme/machines/config_machines.xml [hunk 1]
```diff
@@ -617,16 +617,18 @@
...
          <DESC>ANL/LCRC Linux Cluster</DESC>
          <NODENAME_REGEX>blogin</NODENAME_REGEX>
-         <TESTS>acme_integration</TESTS>
-         <COMPILERS>pgi,intel,gnu,nag</COMPILERS>
-         <MPILIBS>mvapich,mpich,openmpi,mpi-serial</MPILIBS>
-         <CESMSCRATCHROOT>/lcrc/project/$PROJECT/$USER/acme_scratch</CESMSCRATCHROOT>
-         <RUNDIR>$CESMSCRATCHROOT/$CASE/run</RUNDIR>
-         <EXEROOT>$CESMSCRATCHROOT/$CASE/bld</EXEROOT>
+         <TESTS>acme_integration,acme_developer</TESTS>
+         <COMPILERS>gnu,intel</COMPILERS><!--,pgi,intel,nag</COMPILERS>-->
```
Comment (jedwards4b) [inline cime_config/acme/machines/config_machines.xml]: I don't think you need the comments here.

File: cime_config/acme/machines/config_machines.xml [hunk 2]
```diff
@@ -634,16 +636,17 @@
...
          <PROJECT_REQUIRED>TRUE</PROJECT_REQUIRED>
 	 <PROJECT>ACME</PROJECT>
          <batch_system type="pbs" version="x.y">
            <walltimes>
-             <walltime default="true">59:00</walltime>
+             <walltime default="true">15:00</walltime>
            </walltimes>
          </batch_system>
-         <mpirun mpilib="default">
+         <mpirun mpilib="mvapich">
```
Comment (jedwards4b) [inline cime_config/acme/machines/config_machines.xml]: default should have been fine?

**Comment by jedwards4b** [comment]
PR review complete, ready to merge.

**Comment by sarich** [comment]
(Is this a message to me, that I should push to master?)

**Comment by jedwards4b** [comment]
yes

**Pull request merged**